### PR TITLE
Chore: Add Muzli Colors link to colors subcategory and remove khroma file.

### LINKED ISSUE #941: [Add]Muzli Colors to colors subcategory
### Site URL

https://colors.muz.li/

### Description of the above provided site

This is a vibrant hub for endless inspiration, showcasing curated color palettes for your creative projects. Explore, discover, and elevate your projects with ease.

### Category

Frontend

### Sub Category

colors

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [ ] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Oyebolade**
## Fixes Issue
Closes #941

## Changes proposed
I added Muzli Colors link to colors subcategory and removed khroma file.

## Screenshots

Muzlicolors

### CODE REVIEW

**Review by rupali-codes — CHANGES_REQUESTED** [review]
You have made changes in your preview commit, could you please update branch with `linkshub:main`

**Comment by Oyebolade** [comment]
> You have made changes in your preview commit, could you please update branch with `linkshub:main`

Do you mean I should add the message to the PR?
I don't understand you clearly.

**Review by ujjawaltyagii — COMMENTED** [review]
It looks good to me but Ig there was some issue in your initial workflow as when I was trying to see your changes. I was getting this after reloading i saw the real changes-
image

To avoid this : Try making all the changes in your local system first and then make final commit.

**Comment by Oyebolade** [comment]
> It looks good to me but Ig there was some issue in your initial workflow as when I was trying to see your changes. I was getting this after reloading i saw the real changes- image
> 
> To avoid this : Try making all the changes in your local system first and then make final commit.

Thanks @ujjawaltyagii I've pulled the request before I saw the issue.

**Review by CBID2 — DISMISSED** [review]
LGTM! :)

File: database/frontend/colors.json [hunk 1]
```diff
@@ -118,18 +118,18 @@
...
-    "subcategory": "colors"
-  },
   {
     "name": "ColorClaim",
     "description": "ColorClaim is a library of colors that helps developers, designers, photographers, and others to create beautiful and effective color palettes.",
     "url": "https://vanschneider.com/colors",
     "category": "frontend",
     "subcategory": "colors"
+  },
+   {
```
Comment (rupali-codes) [inline database/frontend/colors.json:128]:
```suggestion
  {
```

**Comment by ujjawaltyagii** [comment]
Please resolve merge conflicts !

**Comment by k-deepak04** [comment]
@Oyebolade please resolve the conflicts, so that we can merge this or else let us know if you want to close this one

**Comment by k-deepak04** [comment]
@Oyebolade please resolve the conlflict

**Review by CBID2 — APPROVED** [review]
LGTM! :)

**Review by rupali-codes — APPROVED** [review]
thanks, lgtm

**Pull request merged**